src/Dropdown.js

```diff
@@ -37,17 +37,18 @@ export default class Dropdown extends Component {
     delay: PropTypes.number.isRequired,
     mode: PropTypes.oneOf(['hover', 'click']).isRequired,
     remainTime: PropTypes.number.isRequired,
-    className: PropTypes.string,
     component: PropTypes.node,
+    dropdownLabel: PropTypes.string,
     link: PropTypes.string,
   }
 
   static defaultProps = {
+    dropdownIcon: '',
```

Comment:
`dropdownIcon`, a remainder that we need this props. Could be added once we have SVG issues done.